app/views/categories/index.html.erb

```diff
@@ -0,0 +1,53 @@
+<div class="container">
+    <div id="content" class="space-top-none">
+      <div class="main-content">
+        <div class="space60">&nbsp;</div>
+        <div class="row">
+          <div class="col-sm-3">
+            <ul class="aside-menu">
+                <% @load_categories.each do |category| %>
+                  <li><%= link_to category.name, category %></li>
+                <% end %>
+            </ul>
+          </div>
+          <div class="col-sm-9">
+            <div class="beta-products-list">
+              <h4>Tất cả các sản phẩm </h4>
+              <div class="beta-products-details">
+                <div class="clearfix"></div>
+              </div>
+              <%= will_paginate @products %>
+              <div class="row">
+                <% @products.each do |product| %>
+                  <div class="col-sm-4">
```

Comment:
đoạn div trong for này chị thấy lặp với file `_product_details.html.erb `
thế sao em ko render partial ở đây hì

tức là code trong `div` từ line 22 viết riêng 1 file rồi ở file này và file `_product_details.html.erb ` em chỉ việc render thôi, vừa đỡ foreach vừa đỡ lặp code